pages/api/orders/[orderId]/shipping_products.ts

```diff
@@ -0,0 +1,36 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { bigcommerceClient, getSession } from '../../../../lib/auth';
+
+export default async function shippingAddressesAndProducts(req: NextApiRequest, res: NextApiResponse) {
+    const {
+        query: { orderId },
+        method,
+    } = req;
+
+    try {
+        const { accessToken, storeHash } = await getSession(req);
+        const bigcommerce = bigcommerceClient(accessToken, storeHash, 'v2');
+
+        switch (method) {
+            case 'GET': {
+                const shipping_addresses = bigcommerce.get(`/orders/${orderId}/shipping_addresses`);
+                const products = bigcommerce.get(`/orders/${orderId}/products`);
+
+                const data = await Promise.all([shipping_addresses, products])
+
+                res.status(200).json({ shipping_addresses: data[0], products: data[1] });
```

Comment:
🍹 I don't think we need `Promise.all` here. Couldn't we just await on the get requests? Since we wrap it in a try/catch is should resolve/reject in a similar manner.
```suggestion
                const shipping_addresses = await bigcommerce.get(`/orders/${orderId}/shipping_addresses`);
                const products = await bigcommerce.get(`/orders/${orderId}/products`);

                res.status(200).json({ shipping_addresses, products });
```